File: server/order-service/src/main/java/com/sudo248/invoiceservice/controller/dto/SupplierDto.java
package com.sudo248.invoiceservice.controller.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SupplierDto implements Serializable {
    private String supplierId;
    private String  name, avatar;
}

File: server/product-service/src/main/kotlin/com/sudoo/productservice/controller/dto/SupplierDto.kt
package com.sudoo.productservice.controller.dto

import java.time.LocalDateTime

data class SupplierDto(
    val supplierId: String? = null,
    val userId: String,
    val name: String,
    val avatar: String,
    val brand: String,
    val location: Location,
    val contactUrl: String,
    val locationName: String,
    val totalProducts: Int,
    val rate: Float,
    val createAt: LocalDateTime = LocalDateTime.now(),
)